go.mod

```diff
@@ -7,21 +7,22 @@ require (
 	github.com/ViaQ/logerr v0.0.0-20201008211411-9f0353e42495
 	github.com/coreos/prometheus-operator v0.38.1-0.20200424145508-7e176fda06cc
 	github.com/go-logr/logr v0.2.1
-	github.com/go-openapi/spec v0.19.4
+	github.com/go-openapi/spec v0.19.5 // indirect
 	github.com/google/go-cmp v0.5.0
 	github.com/inhies/go-bytesize v0.0.0-20151001220322-5990f52c6ad6
-	github.com/onsi/ginkgo v1.12.0
-	github.com/onsi/gomega v1.9.0
+	github.com/onsi/ginkgo v1.12.1
+	github.com/onsi/gomega v1.10.1
 	github.com/openshift/api v0.0.0-20200602204738-768b7001fe69
 	github.com/operator-framework/operator-sdk v0.19.4
 	github.com/sergi/go-diff v1.1.0 // indirect
 	github.com/sirupsen/logrus v1.6.0 // indirect
-	gopkg.in/yaml.v2 v2.2.8
+	gopkg.in/yaml.v2 v2.3.0
 	k8s.io/api v0.18.8
 	k8s.io/apimachinery v0.18.8
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/kube-openapi v0.0.0-20200410145947-61e04a5be9a6
-	sigs.k8s.io/controller-runtime v0.6.0
+	k8s.io/kube-openapi v0.0.0-20200410145947-61e04a5be9a6 // indirect
+	sigs.k8s.io/controller-runtime v0.6.2
+	sigs.k8s.io/kustomize/kustomize/v3 v3.5.4 // indirect
```

Comment:
I believe this is a wrong import, could you please ensure one more time `go mod tidy` and `go mod vendor`?